class.leaflet-map.php

```diff
@@ -382,12 +382,21 @@ public function add_popup_to_shape($atts, $content, $shape)
         $message = do_shortcode($message);
         $message = str_replace(array("\r\n", "\n", "\r"), '<br>', $message);
         $message = addslashes($message);
-        $message = htmlspecialchars($message);
+
+        $should_sanitize = apply_filters('leaflet_map_sanitize_popup', true, $message);
```

Comment:
example usage: 

```php
function example_callback( $message ) {
    // don't sanitize popup outputs (maybe dependent on what the $message is)
    return false;
}
add_filter( 'leaflet_map_sanitize_popup', 'example_callback' );
```

See: https://developer.wordpress.org/reference/functions/apply_filters/
And: https://developer.wordpress.org/reference/functions/add_filter/